ChangeLog.d/mbedtls_psa_get_random.txt

```diff
@@ -0,0 +1,8 @@
+Features
+   * Partial implementation of the PSA crypto driver interface: Mbed TLS can
+     now use an external random generator instead of the library's own
+     entropy collection and DRBG code. Enable MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG
+     and See the documentatio of mbedtls_psa_external_get_random() for details.
+   * Applications using PSA crypto can now use its random generator in the
+     mbedtls_xxx API. See the documentation of mbedtls_psa_get_random() for
```

Comment:
Is it even recommended that applications using PSA Crypto continue to call directly into `mbedtls_xxx`? Having this wording in the public changelog makes it look like it's expected behaviour...

Reply:
This is the opposite: applications using `mbedtls_xxx` features (typically X.509/SSL) and calling into PSA. Is the wording here misleading?

Reply:
I think the wording might be too generic, in that case. To me (though my headspace is very much in crypto land and not in SSL) it reads like an implied approval to mix-and-match PSA and mbedtls_xxx for cryptographic functionality.

Reply:
I reworded the second changelog entry. Is it clear now?